testing/ase/test_sw_c_ase.cpp

```diff
@@ -106,54 +104,19 @@ TEST(LibopaecAseOps, ase_ops_01) {
 * @test       ase_ops_02
 *
 * @brief      When the parameters are valid and libopae-ase-c is loaded:
-*             ase_malloc(size_t) function should be called
+*             ase_malloc(size_t) function should return NULL
```

Comment:
Not sure what this is testing.. malloc is failing inside ase_malloc because you've invalidated it with the mock hook.  If it's supposed to fail due to the size being too large, then we shouldn't use invalidate_malloc.

Reply:
Agree. Will change it.